### ISSUE TITLE
cf create-service usage is incorrect:

### ISSUE THREAD

**Issue by Amit-PivotalLabs** [cloudfoundry/cli#704]
```
$ cf -v
cf version 6.13.0-e68ce0f-2015-10-15T22:53:29+00:00
$ uname -a
Darwin Amits-MacBook-Pro.local 15.0.0 Darwin Kernel Version 15.0.0: Wed Aug 26 16:57:32 PDT 2015; root:xnu-3247.1.106~1/RELEASE_X86_64 x86_64
$ cf cs
FAILED
Incorrect Usage. Requires service, service plan, service instance as arguments

NAME:
   create-service - Create a service instance

USAGE:
   cf create-service SERVICE PLAN SERVICE_INSTANCE [-c PARAMETERS_AS_JSON] [-t TAGS]

   Optionally provide service-specific configuration parameters in a valid JSON object in-line:

   cf create-service SERVICE PLAN SERVICE_INSTANCE -c '{"name":"value","name":"value"}'

   Optionally provide a file containing service-specific configuration parameters in a valid JSON object.
   The path to the parameters file can be an absolute or relative path to a file:

   cf create-service SERVICE_INSTANCE -c PATH_TO_FILE

   Example of valid JSON object:
   {
      "cluster_nodes": {
         "count": 5,
         "memory_mb": 1024
      }
   }

EXAMPLE:
   Linux/Mac:
      cf create-service db-service silver -c '{"ram_gb":4}'

   Windows Command Line:
      cf create-service db-service silver -c "{\"ram_gb\":4}"

   Windows PowerShell:
      cf create-service db-service silver -c '{\"ram_gb\":4}'

   cf create-service db-service silver mydb -c ~/workspace/tmp/instance_config.json

   cf create-service dbaas silver mydb -t "list, of, tags"

TIP:
   Use 'cf create-user-provided-service' to make user-provided services available to cf apps

ALIAS:
   cs

OPTIONS:
   -t       User provided tags
   -c       Valid JSON object containing service-specific configuration parameters, provided either in-line or in a file. For a list of supported configuration parameters, see documentation for the particular service offering.
```

The usage pattern describing passing a JSON file doesn't include the `SERVICE` and `PLAN` arguments:

`cf create-service SERVICE_INSTANCE -c PATH_TO_FILE`

The platform-specific usage examples don't include the `SERVICE_INSTANCE` argument:

```
   Linux/Mac:
      cf create-service db-service silver -c '{"ram_gb":4}'

   Windows Command Line:
      cf create-service db-service silver -c "{\"ram_gb\":4}"

   Windows PowerShell:
      cf create-service db-service silver -c '{\"ram_gb\":4}'
```

**Comment by dkoper** [comment]
Hi Amit,

We recently addressed some omissions in the help text of this command; I believe all you have pointed out was resolved in latest release (6.14.0).

Cheers,
Dies Koper
CF CLI PM

**Comment by Amit-PivotalLabs** [comment]
Thanks, new usage looks great!
